onmt/Seq2Seq.lua

```diff
@@ -85,12 +87,32 @@ function Seq2Seq:getOutput(batch)
   return batch.targetOutput
 end
 
+function Seq2Seq:maskPadding(batch)
+  self.models.encoder:maskPadding()
```

Comment:
Shouldn't this line also be  encapsulated by `if self.args.uneven_batches then`?